lib/members.js

```diff
@@ -153,23 +153,23 @@ Membership.evalOverride = function evalOverride(member, change) {
         // it affirms its "aliveness" and bumps its incarnation number.
         member.status = 'alive';
         member.incarnationNumber = +new Date();
-        return _.extend({ type: 'alive' }, member);
+        return member;
     } else if (Membership.isAliveOverride(member, change)) {
         member.status = 'alive';
         member.incarnationNumber = change.incarnationNumber || member.incarnationNumber;
-        return _.extend({ type: 'alive' }, member);
```

Comment:
that's correct. `type` was just transient to communicate what type of update it was. Since I got rid of `new`, there's no longer a need for it. And `new` and `alive` are treated same.